test/ngAnimate/animateCssSpec.js

```diff
@@ -2790,7 +2867,7 @@ describe("ngAnimate $animateCss", function() {
         it("should apply a transition duration if the existing transition duration's property value is not 'all'",
           inject(function($animateCss, $rootElement) {
 
-          ss.addRule('.ng-enter', 'transition: 1s linear color');
+          ss.addRule('.ng-enter', 'transition: 1s cubic-bezier(0.25, 0.1, 0.25, 1) color');
```

Comment:
Browsers have 'ease' as the default transition-timing value, but one browser, IE I think,reports them as cubic-bezier, all others as ease